src/OpenLoco/src/GameCommands/Track/CreateTrack.h

```diff
@@ -14,7 +14,8 @@ namespace OpenLoco::GameCommands
             , rotation(regs.bh & 0x3)
             , trackId(regs.dh & 0x3F)
             , mods(regs.di >> 16)
-            , bridge(regs.edx >> 24)
+            , unkFlags((regs.edx >> 20) & 0xF)
```

Comment:
Any idea what the consequences could have been of these flags having been missing until now? I suppose their purpose is still unknown, given you've labelled them 'unkFlags'...

Reply:
They are only used for ai. Not sure exactly what